.github/workflows/generate.yml

```
@@ -83,6 +83,6 @@ jobs:
           -d '{
             "state":"${{ job.status }}",
             "target_url":"${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}",
-            "description":"${status^}",
+            "description":"'"${status^}"'",
```

```suggestion
            "description":"${{ job.status }}",
```
This was the last thing that worked - was it missing some information we wanted?

---

It was all lowercase and looked unpolished. 🙂